src/MuiForms/EntitySelector.tsx

```diff
@@ -0,0 +1,80 @@
+import Autocomplete from '@mui/material/Autocomplete'
+import CircularProgress from '@mui/material/CircularProgress'
+import TextField from '@mui/material/TextField'
+import * as React from 'react'
+import { useListUsers } from '../../data/user'
+import { EntityKind } from '../../types/interfaces'
+
+interface Entity {
+  kind: string
+  id: string
+  data: any
```

Comment:
Can `data` have a type? I'm guessing `unknown` would be suitable here with the expectation that we test the type as and when we need to manipulate the value.